src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -370,6 +392,7 @@ public class LibraryServiceApi implements AutoCloseable {
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
   public final Shelf createShelf(Shelf shelf) {
+
```

Comment:
It's a line between the validation calls and the request construction code - if there are no validations, then this is what you see. I was fighting another issue where multiple lines were getting squashed onto one line - this is a more palatable outcome.